tsconfig.paths.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "paths": {
+      "@/*": ["src/*"]
+    }
+  }
+}
```

Comment:
Had to define paths in individual file, because CRA was removing it from tsconfig.json with every script execution. (https://github.com/facebook/create-react-app/issues/5118)